broker/src/main/java/org/apache/rocketmq/broker/longpolling/NotifyMessageArrivingListener.java

```diff
@@ -17,21 +17,24 @@
 
 package org.apache.rocketmq.broker.longpolling;
 
-import org.apache.rocketmq.store.MessageArrivingListener;
-
 import java.util.Map;
+import org.apache.rocketmq.broker.processor.PopMessageProcessor;
+import org.apache.rocketmq.store.MessageArrivingListener;
 
 public class NotifyMessageArrivingListener implements MessageArrivingListener {
     private final PullRequestHoldService pullRequestHoldService;
+    private final PopMessageProcessor popMessageProcessor;
 
-    public NotifyMessageArrivingListener(final PullRequestHoldService pullRequestHoldService) {
+    public NotifyMessageArrivingListener(final PullRequestHoldService pullRequestHoldService, final PopMessageProcessor popMessageProcessor) {
```

Comment:
what is pop semantic? Common Queue Operations do not include pop except the special dequeue. Such a name can be confusing. IMO, We should use some common names in the industry. such as poll/pull, peek, push, and so on.